### PR TITLE
Feature/create contributing doc

### PR DESCRIPTION
**Pull request by louise-r-blue**
This is the first PR for the contributing markdown file which includes the code of conduct. Would be good to get this up as the project is now open source. Please check esp the areas (typos and language) before and after the code of conduct.

### CODE REVIEW

**Review by ghostPepperWonton — APPROVED** [review]
Super minor thing you can choose to address: Some of the paragraphs are capped columns (looks like 80 columns), but some are as long as the line you have. There are some plugins to autoformat that and make it consistent. Looks good though!

**Comment by louise-r-blue** [comment]
@danieljauch what do you think is the best/optimal paragraph width?
@danieljauch I added to the editor config but not sure there is any diff.

File: .editorconfig [hunk 1]
```diff
@@ -11,3 +11,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 2
+preferred_line_length = 80
```
Comment (Br3nda) [inline .editorconfig]: i've been using 120 --

80 is a bit of an old standard, and it's rare to find a screen that small now.
:elephant:

Comment (Br3nda) [reply .editorconfig]: We should match our rubocop settings too -- so actually 120 is a better idea i think

**Review by Br3nda — CHANGES_REQUESTED** [review]
editor config as 120 would be better -- or at least match it to the rubocop linter config

**Comment by Br3nda** [comment]
the contributor docs looks fabulous. maybe the editor config change should be on a different PR ?

File: .editorconfig (hunk 1 above)
Comment (louise-r-blue) [reply .editorconfig]: Ok I have removed line 14 from the config.

**Comment by louise-r-blue** [comment]
@Br3nda all done I think

**Review by Br3nda — APPROVED** [review]

**Pull request merged**